website/en/documents/connectors/clickhouse/clickhouse-example.md

```diff
@@ -0,0 +1,85 @@
+# ClickHouse 连接器配置示例
```

Comment:
Should be translated.
Besides,  `website/en/documents/connectors/README.md` should be updated as zh version.